containers/vscode-linux/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/vscode/devcontainers/universal:linux
+
+COPY containers/vscode-linux/requirements.txt /tmp/pip-tmp/
+#COPY requirements.txt /tmp/pip-tmp/
```

Comment:
@reiktar Would it be possible to clean up some of the commented out lines if they are not going to be used?